spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -377,17 +377,20 @@
     let(:response) { described_class.new(success_response, false, config) }
 
     it 'handles an invalid month' do
-      expect(Rails.logger).to receive(:info)
+      expect(Rails.logger).to receive(:info).with(/ALERT ALREADY HAS A VALUE/).exactly(8).times
+      expect(Rails.logger).to receive(:info).with({ event: 'Failure to parse TrueID date' }.to_json).once
```

Comment:
could we make this an `allow` and not worry about the value?
```suggestion
      allow(Rails.logger).to receive(:info)
      expect(Rails.logger).to receive(:info).with({ event: 'Failure to parse TrueID date' }.to_json).once
```

Reply:
Yep, that's a lot better, thank you